app/javascript/mastodon/is_mobile.ts

```diff
@@ -16,9 +16,8 @@ export const layoutFromWindow = (): LayoutType => {
   }
 };
 
-// eslint-disable-next-line @typescript-eslint/ban-ts-comment
-// @ts-expect-error
-const iOS = /iPad|iPhone|iPod/.test(navigator.userAgent) && window.MSStream != null;
+const iOS =
+  /iPad|iPhone|iPod/.test(navigator.userAgent) && window.MSStream != null; // @ts-expect-error MSStream is a legacy vendor property
```

Comment:
Seems like there might be a better way to do this https://stackoverflow.com/a/9039885/455535. It mentions the current way might not work on newer iOS versions, but that's outside the current PR scope